litmus-portal/frontend/src/components/SettingsComponents/UserManagementTab/UserManagement/index.tsx

```diff
@@ -0,0 +1,353 @@
+/* eslint-disable camelcase */
+import {
+  createStyles,
+  FormControl,
+  IconButton,
+  InputAdornment,
+  Menu,
+  MenuItem,
+  Paper,
+  Select,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TextField,
+  Theme,
+  Toolbar,
+  Typography,
+  withStyles,
+} from '@material-ui/core';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import SearchIcon from '@material-ui/icons/Search';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../../../redux/reducers';
+import ButtonFilled from '../../../Button/ButtonFilled';
+import CreateUser from '../CreateUser';
+import EditUser from '../EditUser';
+import useStyles from './styles';
+
+// StyledTableCell used to create custom table cell
+const StyledTableCell = withStyles((theme: Theme) =>
+  createStyles({
+    head: {
+      backgroundColor: theme.palette.common.white,
+      color: theme.palette.common.black,
+    },
+    body: {
+      fontSize: '0.875rem',
+    },
+  })
+)(TableCell);
+
+interface UserData {
+  _id: string; // eslint-disable-line no-eval
+  username: string;
+  email: string;
+  name: string;
+  logged_in: boolean; // eslint-disable-line no-eval
+  created_at: string; // eslint-disable-line no-eval
+  updated_at: string; // eslint-disable-line no-eval
+  removed_at: string; // eslint-disable-line no-eval
+  state: string;
+}
+// UserManagement displays users table
+const UserManagement: React.FC = () => {
+  const classes = useStyles();
+  const [showDiv, setShowDiv] = React.useState<boolean>(false);
+
+  const { userData } = useSelector((state: RootState) => state);
+  // for response data
+  const [rows, setRows] = useState<UserData[]>([]);
+
+  // for filtered row
+  const [selectRow, setSelectRows] = React.useState<UserData[]>(rows);
+
+  useEffect(() => {
+    fetch('http://3.9.117.22:30375/users', {
```

Comment:
This endpoint could be `${config.auth.url}/users`

Reply:
Yeah missed it by mistake